src/uct/api/uct_def.h

```diff
@@ -405,4 +406,62 @@ typedef ucs_status_t (*uct_tag_unexp_rndv_cb_t)(void *arg, unsigned flags,
                                                 const void *rkey_buf);
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Tuning parameters for an UCT endpoint created by @ref uct_ep_create.
+ */
+typedef struct uct_ep_params {
+    /**
+     * Mask of valid fields in this structure, using bits from
+     * @ref uct_ep_params_field. Fields not specified in this mask
+     * would be ignored. Provides ABI compatibility with respect to adding new
```

Comment:
can remove "Provides ABI compatibility with respect to adding new.."

Reply:
"would" -> "will" ?